tests/js/client/authentication/foxx-arango-auth-spec.js

```diff
@@ -172,16 +172,4 @@ describe('Foxx arangoUser', function () {
     expect(result.code).to.equal(200);
     expect(result.body).to.eql(JSON.stringify({user: null}));
   });
-
-  it('should not set the arangoUser object if not authenticated correctly - str obj', function () {
-    const opts = { headers: {
-      authorization: (
-        'Basic ' + new Buffer({}).toString('base64')
```

Comment:
This is a nonsensical test case that relied on our buggy implementation of `Buffer` to return something useful when passed an arbitrary object. This did nothing meaningful and now throws an exception as it should.